v4/source/channels.yaml

```diff
@@ -189,6 +189,42 @@
             // GetPublicChannelsByIdsForTeam
             channels, resp := Client.GetPublicChannelsByIdsForTeam(<TEAMID>, channelIds)
 
+  /channels/{channel_id}/timezones:
+    get:
+      tags:
+        - channels
+      summary: Get users timezones for a channel
+      description: |
+        Get users timezones for a channel.
+        ##### Permissions
+        Must have the `read_channel` permission.
+      parameters:
+        - name: channel_id
+          in: path
+          description: Channel GUID
+          required: true
+          type: string
+      responses:
+        '200':
+          description: Channel timezones retrieval successful
+          schema:
+            $ref: '#/definitions/Timezone'
```

Comment:
Is this the right return? It just returns a single timezone?

Reply:
it return all timezones in that channel, can be 1 or more